src/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/CSharpCodeProvider.cs

```diff
@@ -18,8 +18,12 @@ public CSharpCodeProvider()
             : this(null) {
         }
 
-        // Constructor used for unit test purpose
-        internal CSharpCodeProvider(ICompilerSettings compilerSettings = null) {
+        
```

Comment:
Can you please also update VBCodeProvider?

Reply:
Done

Reply:
Thanks. :)